lib/modules/platform/space/types.ts

```diff
@@ -0,0 +1,169 @@
+export interface SpacePaginatedResult {}
```

Comment:
Use zod schema for types

https://github.com/renovatebot/renovate/blob/main/docs/development/zod.md#L1